user-guide-chapters/introduction/why.adoc

```diff
@@ -65,3 +65,8 @@ and the possible attack surface.
 [14] https://github.com/notaryproject/notation
 
 [15] https://github.com/sigstore/cosign
+
+.MIKE'S QUESTIONS
+****
+. Is this comparison table outdated?
```

Comment:
Maybe the version, but not other items. Mainly because docker distribution has not published a release to fix all this. We should also be comparing against goharbor.